.github/workflows/wasmcloud.yml

```diff
@@ -381,30 +390,23 @@ jobs:
       with:
         path: artifacts
     - run: |
-        mkdir -p ./artifacts/wasmcloud
-        mkdir -p ./artifacts/wash
-
         for dir in ./artifacts/wasmcloud-*; do
-          if [[ "${dir}" == *"-oci" ]]; then # OCI image
```

Comment:
Do we not want to include OCI images in the release artifacts?

Reply:
No, those are meant to be consumed via OCI registries